library_generation/test/utilities_unit_tests.py

```diff
@@ -40,6 +40,8 @@
     library_name="bare-metal-solution",
     rest_documentation="https://cloud.google.com/bare-metal/docs/reference/rest",
     rpc_documentation="https://cloud.google.com/bare-metal/docs/reference/rpc",
+    recommended_package="com.google.example",
```

Comment:
I see that this change is applied to three golden files, is it because this test `LibraryConfig` is used in three tests?

Reply:
@blakeli0 that's right. There are several tests using that one Library config, hence the changes in other golden files

Reply:
OK, ideally we should not reuse test data between unit tests, but let's get this in first and refactor them later.